pkg/apis/sedna/v1alpha1/incrementallearningjob_types.go

```diff
@@ -96,6 +96,11 @@ type InitialModel struct {
 
 type DeployModel struct {
 	Name string `json:"name"`
+	// HotUpdateEnabled defines that the model hot update feature is enabled, the default is disabled.
+	HotUpdateEnabled bool `json:"hotUpdateEnabled,omitempty"`
+	// PollPeriodSeconds is interval in seconds between echo poll of the deploy model config file.
+	// PollPeriodSeconds must be greater than zero and the default value is 60.
```

Comment:
check for `>=0` validation https://book.kubebuilder.io/reference/markers/crd-validation.html

Reply:
fix it